web/draw_layer_builder.css

```diff
@@ -0,0 +1,71 @@
+/* Copyright 2014 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.canvasWrapper {
+  svg {
+    &[data-main-rotation="90"] {
+      transform: none;
```

Comment:
Given that all rotations seem to set this, can this line be perhaps be moved to the `svg`-level to avoid duplicating it multiple times?